services-core/src/solution_submission.rs

```diff
@@ -282,16 +282,16 @@ impl IsOpenEthereumTransactionError for ExecutionError {
 impl IsOpenEthereumTransactionError for Result<(), MethodError> {
     fn is_transaction_error(&self) -> bool {
         match self {
-            Ok(()) => false,
             Err(MethodError { inner, .. }) => inner.is_transaction_error(),
+            _ => false,
```

Comment:
This is just so that this function looks more like the similar function below (next change).